.travis.yml

```diff
@@ -4,17 +4,13 @@ dist: trusty
 
 node_js:
 - '6'
-- '4'
 
-env:
-- PROTAGONIST=true
-- PROTAGONIST=false
-
-before_install:
-- if ! "$PROTAGONIST"; then sudo apt-get purge build-essential && sudo apt-get autoremove; fi
-
-before_script:
-- if ! "$PROTAGONIST" && [ -d "node_modules/protagonist" ]; then echo "Protagonist was successfully installed (should not have been)." && exit 1; fi
+script:
+- if [ -d "node_modules/protagonist" ]; then echo "Protagonist was installed (should not have been)." && exit 1; fi
+- npm test
+- npm i protagonist
+- if ! [ -d "node_modules/protagonist" ]; then echo "Protagonist was not installed." && exit 1; fi
+- npm test
```

Comment:
I think it would make sense to uninstall the drafter.js package before this test run. Otherwise if the code was changed / broken so that Protagonist was never used these tests would still pass due to drafter.js.

Or perhaps we can think about how we can assert that protagonist was actually used for parsing during the tests.